src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -183,6 +183,22 @@
     {% import _self as helper %}
     {% set channel_is_defined = (logs|first).channel is defined %}
 
+    {% if show_level %}
+        <script>
+            function toggleLevel(name) {
+                var rows = document.querySelectorAll('table.logs tr[data-log-level="' + name + '"]');
+                for (var i = 0, row; row = rows[i]; ++i) {
+                    row.style.display = row.style.display == 'none' ? 'table-row' : 'none';
+                }
+            }
+        </script>
+        <div>
+            {% for level in ['INFO', 'NOTICE', 'WARNING', 'ERROR', 'CRITICAL', 'ALERT', 'EMERGENCY'] %}
+                <input type="checkbox" id="log-level-{{ level }}" checked="checked" onclick="toggleLevel('{{ level }}');" />
+                <label for="log-level-{{ level }}">{{ level }}</label>&nbsp;&nbsp;
+            {% endfor %}
+        </div>
+    {% endif %}
```

Comment:
oh, damn, it's copy-pasted...

Reply:
See #24281 for a first step to sanity. But we cant really do something here; as twigbundle and profiler live separated. Causing other issues btw :)

perhaps leverage var-dumper for shared code. Might also help #24151

Im not happy with this :) but also SF problem mostly.